Python/pywarpx/picmi.py

```diff
@@ -1891,6 +1891,7 @@ def add_diagnostic(self):
             self.diagnostic = pywarpx.Diagnostics.Diagnostic(
                 self.name, _species_dict={}
             )
+            self.diagnostic.fields_to_plot = 'none'
```

Comment:
I saw that this PR was revived. I don't think this is the right place for this line since `WarpXDiagnosticBase` is used by all of the diagnostic classes. This setting would mess up `Checkpoint` since `fields_to_plot` shouldn't be set for it. Also, `fields_to_plot` is not used by `ReducedDiagnostic`. The better place for this statement is in `ParticleDiagnostic` after the call to `self.add_diagnostic()`, adding it this way
```
        if 'fields_to_plot' not in self.diagnostic.argvattrs:
            self.diagnostic.fields_to_plot = 'none'
```